app/src/main/java/com/gatchimath/totask/TaskListView.java

```diff
@@ -0,0 +1,79 @@
+package com.gatchimath.totask;
+
+import android.content.Context;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.gatchimath.totask.databinding.TaskBinding;
+
+/**
+ * View for a list of tasks. Each task is rendered as a CardView.
+ */
+public class TaskListView extends RecyclerView {
+	public TaskListView(Context context) {
+		super(context);
+		init(context);
+	}
+
+	public TaskListView(Context context, AttributeSet attrs) {
+		super(context, attrs);
+		init(context);
+	}
+
+	public TaskListView(Context context, AttributeSet attrs, int defStyle) {
```

Comment:
Yes, JavaDoc will automatically inherit docs for overridden methods. http://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#@inheritDoc

This isn't the case for the constructors, but since I don't know what AttributeSet or defStyle are for myself, I don't expect the reader to either, and they can consult RecyclerView for this. These are part of the contract of being an XML-instantiable view and are documented elsewhere.